aten/src/ATen/native/cuda/UnaryOpsKernel.cu

```diff
@@ -63,10 +63,20 @@ void sign_kernel_cuda(TensorIterator& iter){
     }
 }
 
+// overloading erfinv and erfinvf
+template <typename scalar_t>
+__host__ __device__ static inline scalar_t erfinv_wrapper(scalar_t a) {
+  return static_cast<scalar_t>(::erfinvf(static_cast<float>(a)));
+}
+
+__host__ __device__ static inline double erfinv_wrapper(double a) {
+  return ::erfinv(a);
```

Comment:
Where / how is `::erfinv` defined? I'm curious if it does the overloading already.

Reply:
Nice catch! It's in crt/math_functions.hpp, already overloaded, and I have confirmed this by inspecting asm code generated by nvcc. I've modified the PR to use the overloaded function instead.

Reply:
I reverted to the original version again. `half` is not overloaded. If we know better what acc_type means, erfinv_wrapper can be potentially replaced by a cast to acc_type<scalar_t>

Reply:
OK, actually `::erfinv(a)` works for half because `at::Half` defines an implicit conversion to float. I re-appended the overloaded version.